success-stories/index.html

```diff
@@ -45,6 +49,10 @@ <h3>Testimonials</h3>
             </article>
         {% endfor %}
     </div>
+
+    <p>
```

Comment:
Is having the testimonial request twice on the page intentional?

Reply:
@deesprinter pinged me on Slack, and we chatted about this. It is intentional so that people know they can submit the longer form use cases or the shorter testimonials.